Reformat the code to use community best practices

shahinam opened the pull request.

jigarius commented:
I was taking a quick look at the merge request.
- Are we supposed to commit `.rubocop_todo.yml`? Seems like it is a report generated by rubocopy stating all the coding standards which have been disrespected.
The rest of the cleanup looks good and I found the cleaned code to be working. Once we figure out what to do with `.rubocop_todo.yml`, this should be good to merge.

shahinam commented:
I have fixed the merge conflict.

Yes, we will need to add `.rubocop_todo.yml` to progressively fix the issues. I have fixed many issues but left some documentation issues and some which need more understanding of codebase. As we progress we will fix all of them while making sure that we do not introduce new ones.
